src/types.ts

```diff
@@ -26,3 +26,8 @@ export interface IHasNpmConfig {
 export interface IHasSuites {
   suites: Suite[];
 }
+
+export interface IHasNodePath {
```

Comment:
Was never a fan of prefixing interfaces with `I`. Can we get a better name for this? 🙏 

Probably also a good opportunity to clean up the other names as well. `HasNodePath` can likely be shorted to `NodePaths` (plural) or 'NodeConfig'. Same for `HasSuites` etc. which all sound like methods a la `hasSuites()` that ought to return a boolean, rather than interfaces/types.